yb-voyager/src/utils/utils.go

```diff
@@ -130,6 +130,29 @@ func GetSchemaObjectList(sourceDBType string) []string {
 	return requiredList
 }
 
+func ApplyExportSchemaObjectListFilter(allowedObjects *[]string, expectedObjects string) {
+	if expectedObjects == "" {
+		return
+	}
+	expectedObjectsSlice := strings.Split(expectedObjects, ",")
+	newAllowedObjects := []string{}
+	for _, object := range *allowedObjects {
+		if ContainsString(expectedObjectsSlice, object) {
+			newAllowedObjects = append(newAllowedObjects, object)
+		}
+	}
+	*allowedObjects = newAllowedObjects
```

Comment:
Assuming all the names in the `expectedObjects []string` are in upper-case, the entire function can be implemented as:

```
lo.Filter(allowedObjects, func(obj string, _ int) {
  return lo.Contains(expectedObjects, obj)
})
```